temporalio/ui
# Navigate to default namespace from settings

## What was changed


Starts navigating to the default namespace set through Settings 

## Why?


Allows to customize the default namespace to navigate to 

## Checklist


1. Closes 

2. How was this tested:


scenario A: Temporal service is offline (or throws errors)
1. Navigate to both http://localhost:3000/ and http://localhost:3000/namespaces
2. Expected: redirects http://localhost:3000/namespaces/default/workflows
3. Navigate to http://localhost:3000/namespaces/accounting/workflows
4. Expected: should stay on http://localhost:3000/namespaces/accounting/workflows

scenario B: defaultNamespace is set to empty string in settings ``
1. Navigate to both http://localhost:3000/ and http://localhost:3000/namespaces
2. Expected: redirects http://localhost:3000/namespaces/default/workflows
3. Navigate to http://localhost:3000/namespaces/accounting/workflows
4. Expected: should stay on http://localhost:3000/namespaces/accounting/workflows

Scenario C: defaultNamespace is set to `sales`
1. Navigate to both http://localhost:3000/ and http://localhost:3000/namespaces
2. Expected: redirects http://localhost:3000/namespaces/sales/workflows
3. Navigate to http://localhost:3000/namespaces/accounting/workflows
4. Expected: should stay on http://localhost:3000/namespaces/accounting/workflows

3. Any docs updates needed?
